docs/source/tutorials/configure_auto_optimizer.rst

```diff
@@ -0,0 +1,263 @@
+.. _how_to_configure_auto_optimizer:
+
+[BETA] How To Configure Auto Optimizer
+=======================================
+
+Auto Optimizer is a tool that can be used to automatically search Olive passes combination based on:
+
+1. input model
+2. target device
+3. target precision: FP32, FP16, INT8, INT4 and etc.
```

Comment:
>FP32 [](http://example.com/codeflow?start=21&length=4)

I see the line 39 is lower case. should we make it lowe case here?

Reply:
Updated.